### PR TITLE
Fixes Issue 203

### PR DESCRIPTION
**Pull request by achmurali**
This PR fixes Issue #203.

### CODE REVIEW

File: src/mantine-core/src/components/Select/Select.tsx [hunk 1]
```diff
@@ -288,7 +292,7 @@ export function Select({
           aria-autocomplete="list"
           aria-controls={dropdownOpened ? `${uuid}-items` : null}
           aria-activedescendant={hovered !== -1 ? `${uuid}-${hovered}` : null}
-          onClick={() => setDropdownOpened(true)}
+          onClick={() => dropdownOpened ? setDropdownOpened(false) : setDropdownOpened(true)}
```
Comment (rtivital) [inline src/mantine-core/src/components/Select/Select.tsx:295]: `setDropdownOpened(o => !o)`

File: src/mantine-core/src/components/Select/Select.tsx [hunk 2]
```diff
@@ -143,6 +144,10 @@ export function Select({
     inputRef.current?.focus();
   };
 
+  useEffect(() => {
```
Comment (rtivital) [inline src/mantine-core/src/components/Select/Select.tsx:147]: Remove logging

**Review by rtivital — CHANGES_REQUESTED** [review]

**Comment by rtivital** [comment]
@achmurali thanks, it works great, can you also create a same behavior for MultiSelect component with separate PR?

**Comment by achmurali** [comment]
> @achmurali thanks, it works great, can you also create a same behavior for MultiSelect component with separate PR?

Yeah, sure

**Pull request merged**